### PR TITLE
Add DropdownButton

### LINKED ISSUE #1855: Widget catalog seems to be missing DropdownButton?
@mit-mit commented on [Thu Jul 20 2017](https://github.com/flutter/flutter/issues/11319)

I don't see [DropdownButton](https://docs.flutter.io/flutter/material/DropdownButton-class.html) anywhere in https://flutter.io/widgets/ ?

---

@mit-mit commented on [Thu Jul 20 2017](https://github.com/flutter/flutter/issues/11319#issuecomment-316697339)

cc @sethladd 

---

@sethladd commented on [Thu Jul 20 2017](https://github.com/flutter/flutter/issues/11319#issuecomment-316748898)

Thanks for the report!

If someone is interested, this is the file to update: https://github.com/flutter/website/blob/master/_data/catalog/widgets.json

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #1855

### CODE REVIEW

File: src/_data/catalog/widgets.json [hunk 1]
```diff
@@ -235,6 +235,17 @@
     "link": "https://docs.flutter.io/flutter/material/IconButton-class.html",
     "image": "<img alt='' src='https://storage.googleapis.com/material-design/publish/material_v_11/assets/0B_udO5B8pzrzdXVuTlBoOTBjcU0/components_buttons_other1.png'>"
   },
+  {
+    "name": "DropdownButton",
+    "sample": "DropdownButton",
```
Comment (chalin) [inline src/_data/catalog/widgets.json]: I'm not seeing this field being used anywhere. Consider dropping it.

Comment (kwalrath) [reply src/_data/catalog/widgets.json]: The `sample` field is in a bunch of the entries (Scaffold, AppBar), including PopupMenuButton, which I copied. Should I delete it for just this one and/or file an issue to look into the `sample` fields?

Comment (chalin) [reply src/_data/catalog/widgets.json]: Filing an issue is probably the best option, thanks.

Comment (kwalrath) [reply src/_data/catalog/widgets.json]: I'll drop it for this one, since I don't know that a sample exists AND file that issue.

**Pull request merged**